google/container.go

```diff
@@ -51,16 +52,21 @@ func (c *Container) PreSignRequest(_ context.Context, clientMethod stow.ClientMe
 		}
 	}
 
-	headers := make([]string, 0, len(params.Metadata))
-	for k, v := range params.Metadata {
-		headers = append(headers, fmt.Sprintf("x-goog-meta-%s: %s", k, v))
+	headers := make([]string, 0, 1)
+	requestHeaders := map[string]string{"Content-Length": strconv.Itoa(len(params.ContentMD5)), "Content-MD5": params.ContentMD5}
+	if params.AddContentMD5Metadata {
+		headers = append(headers, fmt.Sprintf("x-goog-meta-%s: %s", stow.FlyteContentMD5, params.ContentMD5))
```

Comment:
Can you move the `x-goog-meta-` as a const somewhere in this package... same for other packages